### PR TITLE
#170947595: setup Redux

### PR DESCRIPTION
**Pull request by Kagorora**
#### What does this PR do?

- This PR will setup redux

#### Description of Task to be completed?

- [x] Added Redux packages

- [x] Created redux folder structure

- [x] set redux reducers

#### How should this be manually tested?

- N/A

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [#170947595](https://www.pivotaltracker.com/story/show/170947595)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A

### CODE REVIEW

**Comment by EugeneMunya** [comment]
I think we should structure redux like below
```
 redux/
     actions/
     reducers/
     store/
//OR
actions/
reducers/
store/
```

**Comment by Kagorora** [comment]
> I think the redux folders would be
> 
> ```
>  redux/
>      actions/
>      reducers/
>      store/
> //OR
> actions/
> reducers/
> store/
> ```

@EugeneMunya updated

File: src/index.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ import App from './route/App';
 //   <LandingPage />,
 //   document.getElementById('root')
```
Comment (kagabof) [inline src/index.js:8]: You should not have commented code.

File: src/index.js [hunk 2]
```diff
@@ -7,9 +7,9 @@ import App from './route/App';
...
 
-  ReactDOM.render(
-    <BrowserRouter>
-      <App />,
-    </BrowserRouter>,
-    document.getElementById('root')
+ReactDOM.render(
```
Comment (kagabof) [inline src/index.js:10]: I don't think we can access ours in our components.

**Pull request merged**